contrib/google.golang.org/grpc/grpc.go

```diff
@@ -70,7 +73,7 @@ func finishWithError(span ddtrace.Span, err error, cfg *config) {
 	if errcode == codes.OK || cfg.nonErrorCodes[errcode] {
 		err = nil
 	}
-	span.SetTag(tagCode, errcode.String())
+	span.SetTag(ext.GRPCStatus, errcode.String())
```

Comment:
same comment as in grpc.v12, we need to set the integer code here not the string